readme.txt

```diff
@@ -1,165 +1,200 @@
+
+**BETA SOFTWARE**
+
+This plugin is under active development and, as such, we advise you to first install WooCommerce Admin in a staging/test environment. While this code is being run in production on sites, we recommend you first verify it works for you in a test environment, the same as any plugin for WooCommerce.
+
+**New Reports for WooCommerce**
+
+WooCommerce Admin has a host of new reports that are optimized for speed and have advanced filters that allow you to dive into data about your store:
+
+- Revenue Report
+- Orders Report
+- Products Report
+- Categories Report
+- Coupons Report
+- Taxes Report
+- Downloads Report
+- Stock Report
+- Customers Report
+
+**Customizable Dashboard**
+
+WooCommerce Admin also allows store owners to customize a new dashboard screen with “performance indicators” that correspond in importance to their store’s operation.
+
+== Getting Started ==
+
+Once again: This plugin is under active development and, as such, we advise you to first install WooCommerce Admin in a staging/test environment. While this code is being run in production on sites, we recommend you first verify it works for you in a test environment, the same as any plugin for WooCommerce.
+
+= Minimum Requirements =
+
+* WordPress 5.0
+* WooCommerce 3.6.0 or greater
+* PHP version 5.2.4 or greater. PHP 7.2 or greater is recommended
+* MySQL version 5.0 or greater. MySQL 5.6 or greater is recommended
+
+Visit the [WooCommerce server requirements documentation](https://docs.woocommerce.com/document/server-requirements/) for a detailed list of server requirements.
+
+= Automatic installation =
+
+Automatic installation is the easiest option, as WordPress handles the file transfers and you don't need to leave your web browser. To perform an automatic install:
+1. Log in to your WordPress dashboard.
+2. Go to: Plugins > Add New.
+3. Enter “WooCommerce Admin” in the Search field, and view details about its point release, the rating and description.
+4. Select “Install Now” when you’re ready
+
+= Manual installation =
+
+The manual installation method involves downloading the plugin and uploading it to your web server via your favorite FTP application. The WordPress Codex contains instructions at [Manual Plugin Installation](https://codex.wordpress.org/Managing_Plugins#Manual_Plugin_Installation).
+
+= Where can I report bugs or contribute to the project? =
+
+Bugs should be reported in the [WooCommerce Admin GitHub repository](https://github.com/woocommerce/woocommerce-admin/).
+
+= This is awesome! Can I contribute? =
+
+Yes, you can! Join our [GitHub repository](https://github.com/woocommerce/woocommerce-admin/)
+
+Release and roadmap notes are available on the [WooCommerce Developers Blog](https://woocommerce.wordpress.com/category/wc-admin/)
+
+== Screenshots ==
+
+1. WooCommerce Admin Dashboard
+2. Activity Panels
+3. Analytics
+
+== Changelog ==
+
+= 0.11.0 2019-04-17
+
+- Dev: Extend report submenu items #2033

```

Comment:
I like the idea of this prefix style.  Hopefully we can keep this going forward.

Reply:
I agree @joshuatf it aligns better with how it is done in Woo core, so it makes sense we conform to this structure.